content/en/blog/2024/otel-arrow-production/index.md

```diff
@@ -0,0 +1,413 @@
+---
+title: OpenTelemetry Protocol with Apache Arrow in Production
+linkTitle: OpenTelemetry Arrow in Production
+date: 2024-09-13
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Joshua MacDonald](https://github.com/jmacd) (ServiceNow, Inc), [Laurent
+  Querel](https://github.com/lquerel) (F5, Inc)
+cSpell:ignore: Querel Zstd
+draft: true
+issue: 5193
+sig: OpenTelemetry Arrow
+---
+
+## OpenTelemetry Protocol with Apache Arrow in production
```

Comment:
No need to repeat the title here. (Preview: https://deploy-preview-5198--opentelemetry.netlify.app/blog/2024/otel-arrow-production/)
```suggestion
```